### PR TITLE
Updated project layer, added PreAutorize annotations in controllers

### PR DESCRIPTION
**Pull request by ValentynSharshon**

### CODE REVIEW

**Review by bodya4243 — APPROVED** [review]
all good

File: src/main/java/app/controller/CommentController.java [hunk 1]
```diff
@@ -29,6 +30,7 @@
 public class CommentController {
     private final CommentService commentService;
 
+    @PreAuthorize("hasRole('ROLE_USER')")
```
Comment (EduarfF) [inline src/main/java/app/controller/CommentController.java:33]: As I know, you don't need to add ROLE_

**Review by EduarfF — CHANGES_REQUESTED** [review]
Some comments

File: src/main/java/app/controller/CommentController.java (hunk 1 above)
Comment (VlasenkoYevhenii) [reply src/main/java/app/controller/CommentController.java:33]: One day I've spent a lot of time trying to fix 401 status while the problem was ROLE_ absence =) Let it be)

**Review by VlasenkoYevhenii — APPROVED** [review]

**Review by EduarfF — APPROVED** [review]

**Pull request merged**